vars/getDevTag.groovy

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env groovy
+
+def call() {
+  sh(returnStdout: true, script: '''
+    git fetch origin
+    LATEST_VERSION=$(git tag -l --sort=version:refname | tail -n1)
+    NEXT_VERSION=$(echo $LATEST_VERSION | awk -F. '/[0-9]+\\./{$NF++;print}' OFS=.)
```

Comment:
I don't think we need to higher the version, by appending the number of commits to the latest version we already ensure  that the master is always bigger than the latest, I'm just wondering what happens  if the latest version is something like `0.9.7-rc1`

Reply:
For `0.9.7-rc1` it will output `0.9.8`. IMO bumping the version better describes what the image actually is - dev release of next version. But I'm also ok with keeping latest version. Wdyt?

Reply:
I don't know, I agree that bumping it is better, but at the same time we don't know if the next release is a major, minor or bugfix, anyway are you increasing the bugfix version or the minor?

Reply:
The bugfix/patch/last_number

Reply:
I made up my mind, and I agree with poping up the version, but I would prefer to increase the minor (second second_number)

Reply:
ok, should be fixed now. tried in [1]

[1] https://mobile-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/%20Playground/job/jhellar-multi-test/job/master/13/console